server/server.js

```diff
@@ -142,45 +142,49 @@ Server.prototype._handleSocketMessage = function(socket, data) {
     logging.warn('#socket.message - rejected', socket.id, data);
     return;
   }
+  
+  this._processMessage(socket, message);
+}
+
+Server.prototype._processMessage = function(socket, message) {
+  var messageType = this._getMessageType(message),
+      resource = this._getResource(message, messageType);
 
-  if (!this._messageAuthorize(message, socket)) {
+  if (!this._authorizeMessage(socket, message, messageType)) {
```

Comment:
@vanchi-zendesk - going with "verb first" is fine by me.  It's more of a standard, so I agree with it for the team.